.github/CONTRIBUTING.md

```diff
@@ -13,10 +13,10 @@ Contributions to the code are even more welcome ;)
 
 If you'd like to write some code for seqeralabs/nf-dragen, the standard workflow is as follows:
 
-1. Check that there isn't already an issue about your idea in the [seqeralabs/nf-dragen issues](https://github.com/seqeralabs/nf-dragen/issues) to avoid duplicating work. If there isn't one already, please create one so that others know you're working on this
-2. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the [seqeralabs/nf-dragen repository](https://github.com/seqeralabs/nf-dragen) to your GitHub account
+1. Check that there isn't already an issue about your idea in the [seqeralabs/dragen issues](https://github.com/seqeralabs/dragen/issues) to avoid duplicating work. If there isn't one already, please create one so that others know you're working on this
+2. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the [seqeralabs/dragen repository](https://github.com/seqeralabs/dragen) to your GitHub account
```

Comment:
```suggestion
1. Check that there isn't already an issue about your idea in the [seqeralabs/nf-dragen issues](https://github.com/seqeralabs/nf-dragen/issues) to avoid duplicating work. If there isn't one already, please create one so that others know you're working on this
2. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the [seqeralabs/nf-dragen repository](https://github.com/seqeralabs/nf-dragen) to your GitHub account
```